server/src/main.rs

```diff
@@ -0,0 +1,56 @@
+
+use std::convert::Infallible;
+use std::env;
+use hyper::service::{make_service_fn, service_fn};
+use hyper::{Body, Request, Response, Server};
+use std::collections::HashMap;
+
+fn read_token() -> String {
+    let keys: Vec<&str> = vec![
+    "ethscan",
+    "infura",
+    "blockchain",
+    ];
+
+    let mut token_map = HashMap::<&str, String>::new();
+    for key in keys.into_iter() {
+        let token = match env::var(key) {
+            Ok(token) => token,
+            Err(_) => "".to_string(),
+        };
+        token_map.insert(key, token);
+    }
+
+    serde_json::to_string(&token_map).unwrap()
+}
+
+async fn hello(req: Request<Body>) -> Result<Response<Body>, Infallible> {
```

Comment:
hello?

Reply:
the only simple method to response http request.